src/communication/network_interface_sr.c

```diff
@@ -10820,15 +10820,14 @@ sflashback_get_summary (THREAD_ENTRY * thread_p, unsigned int rid, char *request
   time_t start_time = 0;
   time_t end_time = 0;
 
-  if (flashback_min_log_pageid_to_keep () != NULL_LOG_PAGEID)
+  if (flashback_is_duplicated_request (thread_p))
```

Comment:
If more than two requests come simultaneously, all they pass this condition because no one initializes flashback_Current_conn` yet. It seems to need a latch in the function.

Reply:
Yes, you are right. I will change it